scripts/common_params.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Parse parameters for dependencies or use environment variable
+#
+# List of params is common for all commands:
+# Database: DB_HOST/DB_PORT
+# Bblfshd: BBLFSH_HOST/BBLFSH_PORT
+# Feature extractor: FE_HOST/FE_PORT
```

Comment:
👍 for extracting this logic to re-usable component.

Quick question - as hash, query and report all use it, could you please help me understand what would be the possible downsides of having it implemented directly on JVM application side?

Reply:
Assume that @smacker has just forgot to do it https://github.com/src-d/gemini/issues/120